File: pbweberogwcl/src/app/recuperi/commons/requests/request-modifica-recupero.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { DettaglioRecupero } from '../models/dettaglio-recupero';
export interface RequestModificaRecupero {
  idProgetto?: number;
  recupero?: DettaglioRecupero;
}

File: pbweberog/pbweberog-jar/src/main/java/it/csi/pbandi/pbweberog/integration/request/RequestModificaRecupero.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweberog.integration.request;

import java.io.Serializable;

import it.csi.pbandi.pbweberog.dto.recupero.DettaglioRecupero;

public class RequestModificaRecupero implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private Long idProgetto;
	private DettaglioRecupero recupero;
	public Long getIdProgetto() {
		return idProgetto;
	}
	public void setIdProgetto(Long idProgetto) {
		this.idProgetto = idProgetto;
	}
	public DettaglioRecupero getRecupero() {
		return recupero;
	}
	public void setRecupero(DettaglioRecupero recupero) {
		this.recupero = recupero;
	}
	
	

}
